tools/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+function(add_ram_tool tool_name)
+    add_executable(${tool_name} ${tool_name}.cxx)
+    
+    if(${tool_name} MATCHES "ntuple")
+        target_link_libraries(${tool_name} PRIVATE
+            ramtools_ntuple
+            ROOT::Core
+            ROOT::RIO
+            ROOT::ROOTNTuple
+            ROOT::ROOTNTupleUtil
+        )
+    else()
+        target_link_libraries(${tool_name} PRIVATE
+            ramtools_tree
+            ROOT::Core
+            ROOT::RIO
+            ROOT::Tree
+        )
```

Comment:
The dependencies should come from the outside as `LIBRARY` or something like that. We can use `ROOT_EXECUTABLE`.